hazelcast/src/test/java/com/hazelcast/jet/core/metrics/MetricsTest.java

```diff
@@ -414,6 +417,7 @@ public void availableViaJmx() throws Exception {
         String instanceName = instance().getHazelcastInstance().getName();
         long sum = 0;
         int availableProcessors = Runtime.getRuntime().availableProcessors();
+        System.out.println("Available processors: " + availableProcessors);
```

Comment:
Leftover `System.out.println`, Either change to log or remove.

Reply:
This changes in this class are left by mistake, removed them.